### DISCUSSION TITLE
Memories "since" wording

### DISCUSSION THREAD

**Discussion by dpantel** [immich-app/immich#9067] [General]
Is anyone else bothered by the "X years **since**..." wording of the Memories block?

It's such a small thing, and not "wrong" exactly, but "X years **ago**..." feels so much better to me. Anyone else?

**Comment by mertalev** [comment]
You’re not alone haha: #9025

**Comment by dpantel** [comment]
Aw man. I searched for another discussion or issue for like 10 minutes.
Thanks for the link :)
